src/pkg/packager/lint/lint.go

```diff
@@ -144,9 +143,20 @@ func isPinnedImage(image string) (bool, error) {
 		}
 		return false, err
 	}
+	if isCosignSignature(transformedImage.Tag) || isCosignAttestation(transformedImage.Tag) {
+		return true, nil
+	}
 	return (transformedImage.Digest != ""), err
 }
 
+func isCosignSignature(image string) bool {
+	return (strings.HasSuffix(image, ".sig"))
```

Comment:
Remove the excessive parentheses wrapping the function.

Reply:
🤦‍♂️ good catch, fixed!